omc3/scripts/kmod_summary.py

```diff
@@ -0,0 +1,221 @@
+"""
+Generate summary tables from K-modulation results.
+
+This function collects and summarizes K-modulation results from multiple measurement directories.
+For each beam, it generates:
+1. A DataFrame containing all imported K-modulation measurements.
```

Comment:
Considering we only imported the processed results, not all the modulation data.

```suggestion
1. A DataFrame containing all imported K-modulation measurement results.
```